community-operators/opsmx-spinnaker-operator/opsmx-spinnaker-operator.v1.13.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    certified: "false"
+    containerImage: docker.io/opsmx11/opsmx-spinnaker-operator:v0.1
+    description: Spinnaker is an Open Source, multi-cloud Continuous delivery platform to perform software releases with high velocity and confidence.
+    support: OpsMx
+    createdAt: 2019-04-08T19:49:35Z
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1alpha1","kind":"OpsmxSpinnakerOperator","metadata":{"name":"op-spin"},"spec":{"halyard":{"spinnakerVersion":"1.13.1","image":{"repository":"opsmx11/operator-halyard","tag":"1.18.0"}},"dockerRegistries":[{"name":"dockerhub","address":"index.docker.io","repositories":["library/alpine","library/ubuntu","library/centos","library/nginx"]}],"spinnakerFeatureFlags":["artifacts","jobs"],"minio":{"enabled":true,"imageTag":"RELEASE.2018-06-09T02-18-09Z","serviceType":"ClusterIP","accessKey":"spinnakeradmin","secretKey":"spinnakeradmin","bucket":"spinnaker","nodeSelector":{},"persistence":{"enabled":false}},"rbac":{"create":false},"serviceAccount":{"create":false,"halyardName":"opsmx-spinnaker-operator","spinnakerName":null}}}]'
+  name: opsmx-spinnaker-operator.v1.13.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: OpsmxSpinnakerOperator
```

Comment:
Each CRD requires it's displayName and description be present in this list. Please apply this change to both CSVs

Reply:
Updated all suggested changes.